local-api/formlabs-api-local-openapi.yaml

```diff
@@ -1613,6 +1669,11 @@ paths:
               properties:
                 models:
                   $ref: '#/components/schemas/ModelsSelectionModel'
+                mode:
```

Comment:
Why don't we use the same naming conventions as PreForm? `workspace`?

Reply:
PreFormServer (and the PreFormApi layer) doesn't store a workspace or have a "change workspace" command. Rather, each individual action (orient/layout/support) can be done performed in a way that matches that workspace. Hence the different name.